### DISCUSSION TITLE
Display mobile hamburger menu without items

### DISCUSSION THREAD

**Discussion by Matou0014** [facebook/docusaurus#5392] [Q&A] [answered]
Hi, like I mentioned in the title, is there any way to display the mobile hamburger menu without having anything in the item section of `docusaurus.config.js`?

Currently, my configuration for items is like so:
```
items: [],
```

But on mobile there isn't any menu without having to specify and item making it impossible to navigate my documentation.

**Comment by Josh-Cena** [comment, marked as answer]
Fixed in https://github.com/facebook/docusaurus/pull/5262 and to be released in `2.0.0-beta.5` (the next version)

**Reply by Matou0014** [reply]
Alright, thanks for the info.
